site/docs/master/customize-installation.md

```diff
@@ -42,6 +43,16 @@ By default, `velero install` does not install Velero's [restic integration][3].
 
 If you've already run `velero install` without the `--use-restic` flag, you can run the same command again, including the `--use-restic` flag, to add the restic integration to your existing install.
 
+## Enable features
+
+New features in Velero will be released as beta features behind feature flags which are not enabled by default. These features can be enabled using the `--features` flag to the `velero install` command. This flag takes as value, a comma separated list of feature flags to enable. As an e.g. [CSI snapshotting of PVCs][10] can be enabled using `--features=EnableCSI` flags to the `velero install` command.
+
+Similarly, features may be disabled by removing the corresponding feature flags from the `--features` flag.
```

Comment:
It'd be nice if right here was indicated possible ways to do that.

Reply:
ways to not supply the value to the flag?

Reply:
Sorry, no I mean to disable/remove it.

Reply:
That involves showing how to not pass the value to flag. Not entirely sure how to document that. Suggestions?

Reply:
I think it's the text on LN54. At least that is what I'm talking about :)

Reply:
PTAL at the new edits 😄